code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -1084,3 +1047,44 @@
 /obj/item/bodypart/proc/recolor_external_organs()
 	for(var/obj/item/organ/external/ext_organ as anything in external_organs)
 		ext_organ.inherit_color(force = TRUE)
+
+///A multi-purpose setter for all things immediately important to the icon and iconstate of the limb.
+/obj/item/bodypart/proc/change_appearance(icon, id, greyscale, dimorphic)
```

Comment:
should you be able to change a limb from dimorphic to not?

Reply:
> should you be able to change a limb from dimorphic to not?

You shouldn't be allowed to do most of these things, but for the sake of giving people full control over how limbs appear, its essential.

Say you wanted a robotic chest to appear as a human chest. You need the ability to change it's dimorphism to TRUE to ensure booba shows up on wamen.